### PR TITLE
major additions to "Creating Patterns" section of site

### PR DESCRIPTION
**Pull request by kindohm**
lots of new content. you can preview changes at http://kindohm.github.io/tidalcycles.github.io/patterns.html

these changes are independents of the 0.7 release.

### CODE REVIEW

**Comment by kindohm** [comment]
mainly interested in having someone review this just to make sure I didn't do something foolish with jekyll and core web site stuff.

**Comment by bgold-cosmos** [comment]
Very awesome that you're doing all of this!  It says `scale` only works with positive values, but negative numbers seem to work just fine for me.  Is that out of date?

File: _patterns/bjorklund.md [hunk 1]
```diff
@@ -1,57 +1,72 @@
 ---
-category: bjorklund 
+category: bjorklund
 layout: default
 ---
 
-If you give two numbers in parenthesis after an element in a pattern, then Tidal will
-distribute the first number of sounds equally across the second number of steps:
+If you give two numbers in parenthesis after an element in a pattern, then Tidal
```
Comment (calumgunn) [inline _patterns/bjorklund.md:6]: Should be `parentheses` ;)
NIT-PICKING TYPOBOT HAS POWERED UP

**Comment by calumgunn** [comment]
Looks great to me! Awesome stuff, Mike, thanks so much for taking the time :)

File: _patterns/bjorklund.md (hunk 1 above)
Comment (yaxu) [inline _patterns/bjorklund.md:6]: I reckon parenthesis is right, although to be ultra-pedantic it should probably just be "in brackets" as the concept of parenthesis doesn't really fit here..

Comment (calumgunn) [inline _patterns/bjorklund.md:6]: Very true!

Comment (yaxu) [inline _patterns/bjorklund.md:6]: Fascinatingly, this website disagrees with me.
 http://www.grammarbook.com/punctuation/parens.asp

I'm pretty sure she's totally wrong though. A parenthesis is the phrase, and you can use brackets to denote one.

**Comment by kindohm** [comment]
@bgold-cosmos Just made sure I had 0.7 installed. This works:

```
d1 $ sound "k*8" |=| speed (scale 0.1 2 $ slow 2 sine1)
```

But this does not:

```
d1 $ sound "k*8" |=| speed (scale -2 2 $ slow 2 sine1)
```

It results in an error (which I cannot paste from Atom !@#$):

```
Couldn't match expected type 'Pattern Double' with actual type 'Pattern Double -> f0 (Pattern Double) -> f0 (Pattern Double)' In the first argument of 'speed', namely (not going to type the rest)
```

**Comment by yaxu** [comment]
Try this

```
d1 $ sound "k*8" |=| speed (scale (-2) 2 $ slow 2 sine1)
```

Haskell's a bit surprising around negative numbers.. I think in your version it's actually trying to subtract `2` from `scale`

**Comment by kindohm** [comment]
ah yes, you're right. forgot about the parentheSIS for negatives.

**Comment by calumgunn** [comment]
Was this meant to be closed, @yaxu?

**Comment by yaxu** [comment]
I may have pressed the wrong button

**Comment by kindohm** [comment]
heh. I've also done that many times before. merged it. at least I think you intended to merge it, ha.

**Comment by yaxu** [comment]
The button I was looking for was 'comment' but that's great that you've merged it :)

**Comment by kindohm** [comment]
:open_mouth:

**Comment by lisongx** [comment]
great work Mike!

**Pull request merged**